users/bocaj/bocaj.h

```diff
@@ -16,70 +17,55 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 #pragma once
-#include "quantum.h"
+#include QMK_KEYBOARD_H
```

Comment:
If you have this here, and this file included in the keymap.c files... then you don't need this include in those files.

Reply:
Updates have been applied to the keymap.c files